pkg/cloud/gcp/actuators/machineset/controller.go

```diff
@@ -18,12 +18,13 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/openshift/machine-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
```

Comment:
Use "github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/util", and include in the rest of the imports please